docs/builtins/Operators/and-q.md

```diff
@@ -4,9 +4,9 @@ Use `and?` to apply a logical AND operation to the results of applying a specifi
 
 You can use any data type for the `value` argument as long as the two functions take that same data type and return the resulting boolean value for the logical AND operation performed by the `and?` function.
 
-By convention, the data type <a> is used if an argument represents a type-bound parameter like the `value` argument in this function: 
+By convention, the data type `<a>` is used if an argument represents a type-bound parameter like the `value` argument in this function: 
 
-(defun logicalAnd and?:bool (func1:(<a> -> bool) func2:(<a> -> bool) value:<a>))
+(defun logicalAnd and?:bool (func1:(`<a>` -> bool) func2:(`<a>` -> bool) value:`<a>`))
```

Comment:
I didn't catch this in previous reviews, maybe we should change this to 

```
(defun and?:bool (func1:(<a> -> bool) func2:(<a> -> bool) value:<a>))
```

All in a code block, as in my comment. (That is, with triple tick above, and triple tick below, and no need for ticks inside of the defun).

Reply:
agreed